core-bundle/src/Resources/contao/drivers/DC_Table.php

```diff
@@ -2395,6 +2407,11 @@ public function editAll($intId=null, $ajaxId=null)
 						$blnAjax ? $strAjax .= $this->row($this->strPalette) : $box .= $this->row($this->strPalette);
 					}
 
+					$this->denyAccessUnlessGranted(
```

Comment:
Are we missing a `ReadAction` check when rendering the `editAll` form?

Reply:
I don't think this is needed. Your permission voter is responsible for that.

Reply:
Now I understand, will do.